### ISSUE TITLE
tainted lazarus' flip should have a lower charge + boss room drops should be flippable.

### ISSUE THREAD

**Issue by BadsyEdgecat** [UnderscoreKilburn/repentance-issues#453]
flip being a 6 room charge is fine, and rather strong, on characters who aren't tainted lazarus, because you sometimes get to have extra items. however, when its tied to the main gimmick of the character, only being able to Flip to allow tainted lazarus to get both items when you have access to 6 rooms worth of charge really hurts the character. 

on top of this, tainted lazarus should be able to flip items gained from boss rooms, since those items are always important stat upgrades.

also, just a minor request, but could ghost items not disappear for tainted lazarus?

**Comment by luni3359** [comment]
> also, just a minor request, but could ghost items not disappear for tainted lazarus?

Yes it's really annoying that they fade without any indication but I think they hardcoded pedestals disappearing after leaving a room when they're empty.

**Comment by AyyItsClarkster** [comment]
I agree, Tainted Lazarus is simply weaker than other characters if the two can't use Flip at every opportunity. It's really awkward having 6 charges since you can often be left just short of what you need. I think 4 is better.

**Comment by eyhung** [comment]
I disagree with all these comments.  That's part of the strategy when playing him.  I got no hard mode completion unlocks before the latest change despite several attempts, and all of them fairly quickly once the change was implemented.  6 is enough if you plan things out.

**Comment by eyhung** [comment]
Also, you usually save the boss for the stronger of the two characters, so the weaker one gets the boss item buff.

**Comment by BadsyEdgecat** [comment]
> I disagree with all these comments. That's part of the strategy when playing him. I got no hard mode completion unlocks before the latest change despite several attempts, and all of them fairly quickly once the change was implemented. 6 is enough if you plan things out.

my point is that currently the characters are immensely frustrating just to play for a lot of people, and i think character gimmicks shouldnt be the central point of whats frustrating about a character instead of the difficulty itself

**Comment by BadsyEdgecat** [comment]
hm, should tainted lazarus, with birthright, be able to just grab the ghost item with the side thats ghosted?

**Comment by BadsyEdgecat** [comment]
like, as a way to make birthright also improve flip just by affecting who gets what

**Comment by Rchardon** [comment]
Rep+ heard you for t laz flip
